mono/mini/interp/transform.c

```diff
@@ -6520,6 +6519,16 @@ interp_cprop (TransformData *td)
 				sp [-i].ins = NULL;
 			memset (sp, 0, sizeof (StackContentInfo));
 			sp++;
+		} else if (ins->opcode == MINT_NEWOBJ_FAST && ins->data [0] == 0xffff) {
```

Comment:
we should probably add a define for that magic value (it indicates inlining is involved, right?). can be done in another PR